lib/pack.js

```diff
@@ -21,9 +21,14 @@ class Pack extends BaseCommand {
     return 'pack'
   }
 
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get params () {
+    return ['dry-run']
+  }
+
   /* istanbul ignore next - see test/lib/load-all-commands.js */
   static get usage () {
-    return ['[[<@scope>/]<pkg>...] [--dry-run]']
+    return ['[<@scope>/]<pkg>...]']
```

Comment:
I think the `[[` is relevant here, because it's multiple optional `[<@scope>/]<pkg>` entries.

Reply:
Oops yeah that was a typo from trying to visually grok matching brackets. You'll see it's unbalanced.